torch/_inductor/overrides.py

```diff
@@ -87,7 +88,8 @@ def fuse_fx(gm: torch.fx.GraphModule, example_inputs):
     gm = remove_identity(gm)
     gm = fuse_conv_bn(gm)
     # do mkldnn fusion(conv(linear)+unary(binary)
-    gm = mkldnn_fuse_fx(gm, example_inputs)
+    if not dynamo_config.dynamic_shapes:
+        gm = mkldnn_fuse_fx(gm, example_inputs)
```

Comment:
What's the problem of running `mkldnn_fuse_fx` with dynamic shapes? Fusions should work with both static and dynamic shapes. We can follow up with the fix to enable the dynamic shape. cc @XiaobingSuper

Reply:
The problem is that some of the mkl ops we fuse into don't have meta formulas, like the packing stuff. Whenever you work out the formulas here we can reenable this! However when I looked at it briefly it was not obvious how to fix

Reply:
Thanks for the info, will do a follow-up fix to enable dynamic shapes here.